### DISCUSSION TITLE
Is vercel supporting the manual deployment/release feature?

### DISCUSSION THREAD

**Discussion by happydev000** [vercel/next.js#50024] [Ideas]
### Summary

I'd like to assign 3 domains to one branch (e.g. `main` branch)

- develop.vercel.io (develop env)
- staging.vercel.io (staging env)
- app.vercel.io (production env)

These domains will point only one `main` branch and all updates will be merged to `main` branch while developing. But continuous deployment will work for only `develop` env - `develop.vercel.io`. 
If everything is looking good, I'd like to CLICK the release button (like AWS code pipeline) to apply changes to `staging` env - `staging.vercel.io`.

And same for production.
If everything is looking good in staging, I'd like to CLICK the release button manually to apply changes to `production`.

Is this doable?

**Comment by ozbillwang** [comment]
no this feature yet? anyone can confirm?

**Reply by happydev000** [reply]
not yet. never got any replies or responses
